resources/dicts/events/ceremonies/ceremony_text.json

```diff
@@ -800,38 +800,91 @@
+            ],
+            "righteous": [
+                "(prefix)paw says a silent prayer to StarClan as they get their medicine cat name of m_c. StarClan protect them, StarClan protect all of them, they hope they'll receive guidince in the moons ahead."
+            ],
+            "shameless": [
+                "(prefix)paw blurts out their new medicine cat name of m_c, cutting off the words of the ceremony midmeow, not suprising anyone. There's no arguing with m_c, so the other medicine cats chear for them, through purrs of amusement."
+            ],
+            "sneaky": [
+                "(prefix)paw's eyes glint as they get the medicine cat name of m_c. No one seems to notice that they don't agree to all of the vows of being a medicine cat, instead only nodding at some, and agreeing to others with a meow."
+            ],
+            "strange": [
+                "No one knows why (prefix)paw brought a mossball to their medicine cat naming ceremony, but StarClan grants them their name of m_c all the same, welcoming them into the full mysteries of StarClan."
+            ],
+            "strict": [
+                "The way (prefix)paw is sitting is so stiff you could use them as reenforcement of the camp walls. Even when they get their medicine cat name of m_c, they don't loosen up, and only seem to sit up even more."
+            ],
+            "thoughtful": [
+                "Even though this is their medicine cat naming ceremony, (prefix)paw can only think about their patients back at camp, and asks if they can return early after they get their name of m_c.",
+                "StarClan honors the way (prefix)paw always goes the extra step to do their best as a medicine cat, and gives them the name of m_c, knowing that m_c will continue to do their best."
+            ],
+            "troublesome": [
+                "This is a serious time for (prefex)paw, their medicine cat naming ceremony! Naturally, they're cracking jokes and being irreverent right up until the moment they get their name of m_c, and quiet down long enough to think of something funny to add."
+            ],
+            "vengeful": [
+                "The other medicine cats mistake it as being calm, but (prefex)paw isn't quiet because they're at peace, but because they are angry. Every snide remark, every time someone sneered at them, every time they were treated less than, all of it goes through their mind. Now that they are m_c, c_n better treat them with respect, or there would be consequences."
+            ],
+            "wise": [
+                "Medicine cat training had just felt right for (prefix)paw, more like remembering then learning, like they had done this before. When it comes time for their naming, they ask to name themselves, and chose m_c for their name. It feels right for them to be called m_c."
```

Comment:
I like it! Just vague enough to not be specifically about reincarnation, but suggesting it